### ISSUE TITLE
Redirect from datproject.org to dat.foundation

### ISSUE THREAD

**Issue by martinheidegger** [datproject/public-events#17]
Going to https://events.datproject.org shows the site locally, it should probably better be redirecting to https://events.dat.foundation. Maybe a netlify redirect should suffice?

https://docs.netlify.com/routing/redirects/redirect-options/#domain-level-redirects

**Comment by ssaavedra** [comment]
I agree, do we want to cache the redir. in the browser? (301 v. 303)

**Comment by ssaavedra** [comment]
Maybe they weren't working because the redirect was going from https to http? I opened a PR but this needs to be tested through Netlify to see if that was the issue

**Comment by martinheidegger** [comment]
If netlify doesn't work, maybe we can make javascript redirects? (checking the domain and if its wrong: redirecting to events.dat.foundation?
